src/setIntersection.js

```diff
@@ -0,0 +1,15 @@
+export default function setInstersection(array, intersect){
+  let result = []
+  for(let i=0;i<array.length;i++){
```

Comment:
add spaces around the `=`
so `i = 0; i < array.length; i++`
make it as close to english syntax as you can.